pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -110,6 +116,23 @@ func (p *perfCounterNodeStatsClient) startMonitoring() error {
 		p.collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter, networkAdapterCounter)
 	}, perfCounterUpdatePeriod)
 
+	// Cache the CPU usage every 15 seconds
+	go wait.Forever(func() {
```

Comment:
Yeah that's right, the update frequency is different in here.

And for the cpuUsageNanoCores it could have at most 14 seconds lag of the real value, as the goroutine runs every 15 seconds and if we query at the 29th second it will still return the value at the 15th second.